client/src/components/App.js

```diff
@@ -34,11 +34,9 @@ const App = (props) => {
         <Route exact path="/" component={HomePage} 
         />
         <Route 
-        exact path="/restaurants" component={RestaurantList} 
-        />
+        exact path="/restaurants" component={RestaurantList} />
         <Route exact path="/restaurants/:id"
-          render={(props) => <RestaurantShow {...props} currentUser={currentUser} />}
-        />
+          render={(props) => <RestaurantShow {...props} currentUser={currentUser} />} />
```

Comment:
you have multiple patterns here for designating the Routes - prefer separate lines for each prop attribute for readability of all Routes:
```js
<Route
  exact
  path="/"
  render={(props) => ... }
/>
```
the opening and closing tags on their own lines, and each prop on its own line